### DISCUSSION TITLE
Anh Phúc Done Task Giao diện #1

### DISCUSSION THREAD

**Discussion by nvatuan** [nvatuan/PBL6#3] [Show and tell]
@xuanphuc97 Anh Phúc check Done cái task https://github.com/nvatuan/PBL6/projects/2#card-69464291 rồi, anh up ảnh demo vào thread này cho mọi người coi cái nào.

**Comment by xuanphuc97** [comment]
![image](https://user-images.githubusercontent.com/64199092/135606952-abdbfd73-129a-4fff-a13f-826163992ea4.png)
![image](https://user-images.githubusercontent.com/64199092/135606720-c238802b-21fd-46f2-aa2f-0aab0d9eae72.png)
![image](https://user-images.githubusercontent.com/64199092/135606776-9e618779-d463-4723-8883-cbc33e973481.png)
![image](https://user-images.githubusercontent.com/64199092/135606871-650e9baf-7fc3-491c-8871-cd2853217c8d.png)
